cfbs/commands.py

```diff
@@ -460,5 +464,64 @@ def install_command(destination=None) -> int:
     cp("out/masterfiles", destination)
     return 0
 
+
 def help_command():
-    pass # no-op to allow help to be printed
+    pass  # no-op to allow help to be printed
+
+
+def info_command(modules, index=None):
+    if not index:
+        index = get_index_from_config()
+    index = Index(index)
+    build = get_definition()["build"]
+    alias = None
+
+    for module in modules:
+        print()  # whitespace for readability
+        if not index.exists(module):
+            print("Module '{}' does not exist".format(module))
+            continue
+        if module in index:
+            data = index[module]
+            if "alias" in data:
+                alias = module
+                module = data["alias"]
+                data = index[module]
+            found = [m for m in build if m["name"] == module]
+            data["status"] = "Added" if len(found) > 0 else "Not Added"
+        else:
+            if not module.startswith("./"):
+                module = "./" + module
+            data = [m for m in build if m["name"] == module]
+            if len(data) != 1:
+                print("Path {} exists but is not yet added as a module.".format(module))
+                continue
+            data = data[0]  # take the first (and only) item
+            data["status"] = "Added"
+        ordered_keys = [
+            "module",
+            "version",
+            "status",
+            "by",
+            "tags",
+            "repo",
+            "commit",
+            "dependencies",
+            "added_by",
+            "description",
+        ]
+        data["module"] = module + "({})".format(alias) if alias else module
```

Comment:
Make sure the `if alias else module` actually applies to the part of the expression that you want it to apply.